### PR TITLE
docs: Workspace refactor

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR refactors the workspace section. 

## Changed Pages

💻 [Workspaces](https://deploy-preview-5064--docs-spectrocloud.netlify.app/workspace/)

## Jira Tickets

🎫 [Jira Ticket]()

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

### CODE REVIEW

File: docs/docs-content/workspace/workspace.md [hunk 1]
```diff
@@ -8,56 +8,51 @@ sidebar_custom_props:
...
+- **Resource Quotas**: Defining and enforcing CPU and memory usage limits within namespaces, applied uniformly across
+  all clusters in the workspace.
 
-Role bindings and cluster role bindings are specified within workspaces. Furthermore, these role bindings and cluster
-role bindings are created in every cluster within the workspaces, thus enabling centralized RBAC.
+## Centralized Access Control
 
-## Utilization
+Workspaces simplify Role-Based Access Control (RBAC) by centralizing management of role bindings and cluster role
+bindings. You can specifying role bindings and cluster role bindings within the workspace to automatically apply them to
```
Comment (caroldelwing) [inline docs/docs-content/workspace/workspace.md:34]:
```suggestion
bindings. You can specify role bindings and cluster role bindings within the workspace to automatically apply them to
```

File: docs/docs-content/workspace/workspace.md [hunk 2]
```diff
@@ -8,56 +8,51 @@ sidebar_custom_props:
...
+- **Cost Attribution**: Calculating costs for workloads based on resource utilization, enabling internal charge-back or
+  show-back for teams or applications.
 
-## Workload Visibility
+## Backup and Disaster Recovery
 
-Workspaces provide a workload browser to view all the workloads such as pods, deployment, jobs, stateful sets, etc.,
-deployed in all the namespaces that are part of the workspace across all the clusters. The workload browser aggregates
-resources across clusters from relevant namespaces and presents them with centralized visibility.
+**Workspace-Based Backup**: extends cluster-level backups to include namespaces in all clusters within a workspace. For
```
Comment (caroldelwing) [inline docs/docs-content/workspace/workspace.md:50]:
```suggestion
**Workspace-Based Backup**: Extends cluster-level backups to include namespaces in all clusters within a workspace. For
```

**Review by caroldelwing — APPROVED** [review]
Approved with a few suggestions

File: docs/docs-content/workspace/workspace-mgmt/delete-workspace.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+sidebar_label: Delete Workspace
+title: Delete Workspace
+description: "Learn how to restrict certain images from being used by your workspace"
```
Comment (lennessyy) [inline docs/docs-content/workspace/workspace-mgmt/delete-workspace.md:4]:
```suggestion
description: "Learn how to delete a workspace. "
```

**Pull request merged**